CDP4IMEInstaller/CDP4RelationshipMatrix.wxs

```diff
@@ -18,6 +18,26 @@
         <File Id="fil_RelationshipMatrix_csvhelper" Vital="yes" Source="$(var.CDP4RelationshipMatrix.TargetDir)CsvHelper.dll" KeyPath="yes" />
       </Component>
 
+	  <Component Id="comp_RelationshipMatrix_AsyncInterfaces" Guid="{BD8B9F2C-2BF7-4309-9322-BEA012C5ABDD}" Location="local" Win64="$(var.Win64)">
+		<File Id="fil_RelationshipMatrix_AsyncInterfaces" Vital="yes" Source="$(var.CDP4RelationshipMatrix.TargetDir)Microsoft.Bcl.AsyncInterfaces.dll" KeyPath="yes" />
```

Comment:
i propose we add these dependencies in the `Dependencies.wxs` file instead of here. These should not be copied to the plugin folder but to the main bin folder.